src/ring_theory/hahn_series.lean

```diff
@@ -797,14 +797,19 @@ variables {Γ' : Type*} [ordered_cancel_add_comm_monoid Γ']
         (function.embedding.prod_map ⟨f, hfi⟩ ⟨f, hfi⟩),
         (emb_domain ⟨⟨f, hfi⟩, hf⟩ x).coeff (ij.1) *
         (emb_domain ⟨⟨f, hfi⟩, hf⟩ y).coeff (ij.2),
-      { simp },
+      { dsimp only [rel_embedding.coe_fn_mk],
```

Comment:
this extra bit was truly hellish to find, `dsimp` was doing something, `squeeze_dsimp` was telling me it was something else, `show_term { dsimp }`  was saying it was `refine id _`... so done